pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -315,6 +315,20 @@ const (
 	DefaultNodeVolumeGP3IOPS = 3000
 )
 
+type IPFamily string
+
+const (
```

Comment:
This will let the schema generation code list supported values for the IPFamily field:

```suggestion

// Values for `IPFamily`
const (
```

Reply:
Oh cool, I didn't know that.